operators/admin/olm-configuring-proxy-support.adoc

```diff
@@ -29,3 +29,12 @@ endif::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
 
 include::modules/olm-overriding-proxy-settings.adoc[leveloffset=+1]
 include::modules/olm-injecting-custom-ca.adoc[leveloffset=+1]
+
+ifndef::openshift-dedicated,openshift-rosa,openshift-rosa-hcp[]
+[role="_additional-resources"]
```

Comment:
Per the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies), I think assembly-level Additional Resources sections need an anchor ID:
```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
```